etc/p3-config/p3-config.yml

```diff
@@ -424,7 +439,7 @@ p3_image_fedora:
 
 # Build of OpenHPC on CentOS
 p3_image_openhpc:
-  name: "CentOS7-OpenHPC"
```

Comment:
we might need to think more about renames, maybe we should delete or hide the image with the old name

Reply:
So far I've renamed the previous image to (say) CentOS7-20171004 when a new image is rotated in.  Is that sensible?  And when should the old images be deleted?  I guess we'd need a check that no users had live instances derived from it, at the least.